Write a custom changelog describing certain upstream changes.
Index: hyphen-ru-20030310/changelog
===================================================================
--- /dev/null	1970-01-01 00:00:00.000000000 +0000
+++ hyphen-ru-20030310/changelog	2013-05-13 11:47:52.000000000 +0400
@@ -0,0 +1,18 @@
+2003-03-10  Alexander I. Lebedev  <swan@scon155.phys.msu.su>:
+	- Fixed occurrences in about 24,000 vocabulary words
+	- Released under the terms of the LaTeX project public license,
+	  version 1.2 or higher
+
+2001-08-02  Alexander I. Lebedev  <swan@scon155.phys.msu.su>:
+	- Second release. Patterns generated from scratch with patgen from a
+	  990,000-word list and then manually corrected
+	- Released under restrictive license terms, with free distribution and
+	  reencoding as the only permitted modification
+
+2000-10-09  Alexander I. Lebedev  <swan@scon155.phys.msu.su>:
+	- Fixed occurrences in over 51,000 vocabulary words
+
+2000-01-04  Alexander I. Lebedev  <swan@scon155.phys.msu.su>:
+	- Original release based on Dimitri Vulis' ruhyphdv patterns package,
+	  released into the public domain
+	- Fixed occurrences in about 67,000 vocabulary words
